examples/textConnect/contents/setting.html

```diff
@@ -4,210 +4,218 @@
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select15" id= "select15">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-    </div>
-    
-　　</div>
-  <br >
-  <div class="block">
-    <label class="kintoneplugin-label">
-        <span id ="container_label">項目間の記号</span>
-    </label><br >
-    <div class="kintoneplugin-desc">結合される項目の間に表示される記号を入力してください。<br >未選択の場合、各項目が直接結合されます。</div>
-    <div class="kintoneplugin-row">
-        <span class="num">①</span>
-        <div class="kintoneplugin-input-outer">
-            <input name="between" id= "between1" class="kintoneplugin-input-text" type="text">
-        </div> 
-    </div>
-    
-    <div class="kintoneplugin-row">
-        <span class="num">②</span>
-        <div class="kintoneplugin-input-outer">
-            <input name="between" id= "between2" class="kintoneplugin-input-text" type="text">
-        </div> 
-    </div>
-    
-    <div class="kintoneplugin-row">
-        <span class="num">③</span>
-        <div class="kintoneplugin-input-outer">
-            <input name="between" id= "between3" class="kintoneplugin-input-text" type="text">
-        </div> 
-    </div>
-    
-    </div>
-  <br >
-  <div class="block">
-    <label class="kintoneplugin-label">
-        <span id ="container_label">結合された文字列を表示する項目</span>
-        <span class="kintoneplugin-require">*</span>
-    </label><br >
-    <div class="kintoneplugin-desc">結合された文字列を表示する項目を選択してください。</div>
-    <div class="kintoneplugin-row">
-        <span class="num">①</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="copyfield" id= "copyfield1">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>  
-    </div>
-    <div class="kintoneplugin-row">
-        <span class="num">②</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="copyfield" id= "copyfield2">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>  
-    </div>
-    <div class="kintoneplugin-row">
-    <span class="num">③</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="copyfield" id= "copyfield3">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>  
-    </div>
-  </div>
-  <br >
-  <div class="block">
-    <button type="button" id="submit" class="kintoneplugin-button-dialog-ok">     保存する     </button>
-    <button type="button" id="cancel" class="kintoneplugin-button-dialog-cancel">     キャンセル   </button>
   </div>
-</div>  
\ No newline at end of file
+</script>
```

Comment:
Please remove script tag and convert it into a div tag along with line 11

Reply:
Updated the codes. Please see the latest commit made on 8/17.